dist.ini

```diff
@@ -1,70 +1,12 @@
 name    = Math-Business-Lookback
-author  = binary.com <BINARY@cpan.org>
-license = None
-copyright_holder = binary.com
-copyright_year   = 2017
-main_module = lib/Math/Business/Lookback.pm
-[Git::GatherDir]
-exclude_filename = Makefile.PL
-include_dotfiles = 1
-[PruneCruft]
-except = t/rc/\.perl.*rc$
-[ManifestSkip]
-[MetaYAML]
-[License]
-[Readme]
-[MakeMaker]
-eumm_version = 6.48
-prereq_fatal = 1
-[ExecDir]
-[ShareDir]
-dir = share
-[Manifest]
-[TestRelease]
-[ConfirmRelease]
-[Prereqs::FromCPANfile]
-[Prereqs / BuildRequires]
-perl = 5.014000
-[CheckPrereqsIndexed]
-[CheckExtraTests]
-[VersionFromModule]
-[ReversionOnRelease]
-[NextRelease]
-[OurPkgVersion]
-[InstallGuide]
-[MetaJSON]
-[InsertExample]
+author  = binary.com <binary@cpan.org>
```

Comment:
i have noticed that the author email for all the other repos have been BINARY@cpan.org.

Reply:
You are right. Thanks